bitcoin/src/crypto/key.rs

```diff
@@ -62,6 +62,10 @@ impl XOnlyPublicKey {
             .map_err(|_| ParseXOnlyPublicKeyError::InvalidXCoordinate)
     }
 
+    /// Returrns the wrapped generic Sec256k1 x-only public key
```

Comment:
```suggestion
    /// Returns the wrapped generic Sec256k1 x-only public key.
```

Typo in 'Returns' and also a full stop because [code] cleanliness is close to godliness.